jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -479,24 +506,65 @@ def __init__(self, client: Client, **kwargs):
                 f"Could not create mount dir {PFS_MOUNT_DIR}. Try setting the PFS_MOUNT_DIR env var."
             )
             raise e
-        super().__init__(**kwargs)
 
-    # TODO: don't ignore name in the input spec
-    # right now, when a repo is mounted as part of mounting datum(s), the
-    # toplevel directory for the repo is project_repo_branch. we should
-    # update this to respect the name in the pfsinput objects being supplied
-    # to the mount_datums api. however, this is not that simple as the name
-    # can't simply be looked up in the datuminfo. we will need to have some
-    # mapping to track which files within a datum belong to which name.
+    def _populate_name_table(self, input: pps.Input):
+        """
+        Given the input spec, populates the bi-directional name to commit_uri
+        mapping of the commits mounted by the input.
+        """
+        if input.pfs:
+            repo = pfs.Repo(
+                name=input.pfs.repo,
+                project=pfs.Project(
+                    name=input.pfs.project if input.pfs.project else "default"
```

Comment:
You don't need to change this, but I think we've talked about this in/around code reviews before. This is the situation where python's `or` semantics are nice. You are able to make lines like this more succinct with `input.pfs.project or "default"` which is equivalent what you have here.